crates/ruff_python_formatter/resources/test/fixtures/ruff/docstring_chaperones.py

```diff
@@ -0,0 +1,51 @@
+class Sample:
+    """Needs chaperone\\" """
+
+    """Needs chaperone\\\ """
```

Comment:
This file doesn't test the chaperon handling except for the first string because all subsequent strings aren't docstrings.